packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -285,45 +315,50 @@ const menuPortalStyles = (props: TProps) => (base: TBase) => ({
   zIndex: props.menuPortalZIndex,
 });
 
-const multiValueStyles = () => (base: TBase) => {
-  return {
-    ...base,
-    height: designTokens.sizeHeightTag,
-    backgroundColor: designTokens.backgroundColorForTag,
-    padding: '0',
+const multiValueStyles =
+  (props: TProps & { isNewTheme: boolean }) => (base: TBase) => {
+    return {
+      ...base,
+      height: designTokens.sizeHeightTag,
+      backgroundColor: designTokens.backgroundColorForTag,
+      padding: '0',
+      border: props.isNewTheme && `1px solid ${designTokens.colorNeutral85}`,
```

Comment:
In case `props.isNewTheme` is _false_, the `border` value would be _'false'_. I guess we would want something like this:
```suggestion
      border: props.isNewTheme ?? `1px solid ${designTokens.colorNeutral85}` : 'none',
```